README.md

```diff
@@ -35,101 +39,45 @@ As a Maker
-
-Some useful resources:
-**DataMapper**
-- [Datamapper wiki](https://en.wikipedia.org/wiki/DataMapper)
-- [Sinatra, PostgreSQL & DataMapper recipe](https://github.com/sinatra/sinatra-recipes/blob/master/databases/postgresql-datamapper.md)
+- I need 2 tables: one for users, one for peeps.
+- Importantly, the column user_id will link them.
+- Unique keys (user_id) from the user table will be associated
+with all peeps ever made in the peeps table
 
-**Ruby Object Mapper**
-- [ROM](https://rom-rb.org/)
+| user_id |     email         | name      | handle      | password |
+|---------|-------------------|-----------|-------------|----------|
+| 1       | william@gmail.com	| bill      | billy123    | bad_pw   |
+| 2       |	jessica@gmail.com | jess      | knock_knock | vbad_pw  |
 
-**ActiveRecord**
-- [ActiveRecord ORM](https://guides.rubyonrails.org/active_record_basics.html)
-- [Sinatra, PostgreSQL & ActiveRecord recipe](http://recipes.sinatrarb.com/p/databases/postgresql-activerecord?#article)
 
-Notes on functionality:
-------
+| peep_id |  content          | time_stamp            | user_id |  
+|---------|-------------------|-----------------------|---------|
+| 1       | 'hi @billy123'    |  2021-04-25 20:26:10  | 2       |
+| 2       |	'hi @knock_knock' |  2021-04-25 20:27:00  | 1       |
+| 3       | 'who's there?'    |  2021-04-25 20:32:24  | 2       |
+| 4       |	'uh..'            |  2021-04-25 20:42:44  | 1       |
 
-* You don't have to be logged in to see the peeps.
-* Makers sign up to chitter with their email, password, name and a username (e.g. samm@makersacademy.com, password123, Sam Morgan, sjmog).
-* The username and email are unique.
-* Peeps (posts to chitter) have the name of the maker and their user handle.
-* Your README should indicate the technologies used, and give instructions on how to install and run the tests.
+in the above example, user 1 (bill) made peeps 1 and 3, and user 2(jess) made peeps 2 and 4.
 
-Bonus:
------
+### CLASSES ###
 
-If you have time you can implement the following:
+Class USER should have attributes of email, username, handle, password, id.
+Class USER should make instances of Class PEEP.
+Class USER should also be able to sign up - so may need a self.create method.
 
-* In order to start a conversation as a maker I want to reply to a peep from another maker.
+Class PEEP should have attributes of id, content, time stamp, associated user.
+Class PEEP should have Class Methods including self.all and self.create.
 
-And/Or:
+USER.MAKE_PEEP should prompt PEEP.SELF.CREATE
 
-* Work on the CSS to make it look good.
 
-Good luck and let the chitter begin!
+### VIEWS/USER JOURNEY ###
 
-Code Review
------------
+One view each for post_cheep, view_all_cheeps and signup. View_all_cheeps and post_cheer should be mutually accessible, but the user should begin on signup which (when successful) will reroute to view_all_cheeps.
```

Comment:
I really liked that you added an explanation of the views from the user's perspective to your README file. 
I found the route names confusing for a few seconds (cheep, instead of peep- did I miss something?), as they appear differently on the app (post_cheer == new_peep).